### ISSUE TITLE
[BUG] Full screen is not working on the reader

### ISSUE THREAD

**Issue by jayjoshi64** [stumpapp/stump#355]
**Describe the bug**
Clicking on full screen icon does nothing. 

**To Reproduce**
1. Open any ebooks (I am using .epub book)
2. Click on the full screen. 
3. ...Nothing happens

**Expected behavior**
The reader view should be full screened 

**Screenshots**
![Jun-26-2024 12-54-03](https://github.com/stumpapp/stump/assets/20963493/d297847b-32b0-437f-add5-9ba3bfd86712)

**Desktop (please complete the following information):**

- OS: macOS
- Browser: Edge & Chrome

**Comment by aaronleopold** [comment]
This isn't actually a bug, just perhaps misleading and/or bad UX.

The header and footer UI components are fixed and constantly visible. When you enter "fullscreen" mode they instead hide after a second or two of losing focus / mouse exit.

It does not control your browser's full screen functionality.

**Comment by jayjoshi64** [comment]
I see.  Let's make it a little obvious if possible.   

- tbh, the header is so small, I don't think this makes that much sense in desktop version.  But for phone, it might be good to have. 
- adding some reaction would be helpful. For example, instead of waiting for the mouse to exit the component, we can immediately hide the menu.  And instead, we can add a small down arrow which use can click to unhide it.  Many web apps use such feature. 
- also, the button is unresponsive. Changing the icon a little bit can also help.  But this would be secondary.

**Comment by jayjoshi64** [comment]
Although, it's not the most important thing probably. I'll see if I can raise a PR in few weeks.

**Comment by aaronleopold** [comment]
Yeah for sure, I'm down to play with the UI to improve the overall experience. I won't have time in the immediate few weeks, as I am focusing on a few other features already, but can make some time afterwards.

> I'll see if I can raise a PR in few weeks.

If you need anything please reach out! There is a dedicated dev channel on the Discord, as well.

**Comment by aaronleopold** [comment]
I'm going to close this and fold the UI suggestions into https://github.com/stumpapp/stump/issues/378
